### ISSUE TITLE
WISH

### ISSUE THREAD

**Issue by ivan108** [UCSF-Costello-Lab/LG3_Pipeline#58]
add real time log, to check what steps are already completed

**Comment by HenrikBengtsson** [comment]
We got `lg3 status`.  Are you thinking of finer progress/status updates?

**Comment by ivan108** [comment]
Yes, e.g. steps of Recal. Basically same info we get after the run in *.out and *.err files but in real time (stdout and stderr combined). When run is finished, real time log can be deleted, since we will get  *.out and *.err...
